scripts/sign_bsta.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```

Comment:
Have you tested this script in both python2 and python3?

Reply:
I have now, and it seems to work. :-)

Reply:
Okay then :-)